models/post.model.js

```diff
@@ -10,7 +10,11 @@ const postSchema = new mongoose.Schema({
         type: String,
         required: true
     },
-    class: {
+    grade: {
+        type: String,
+        required: true
+    },
+    classname: {
```

Comment:
we should use like className as swm htet is already developing the class with className: blah blah